release notes/v0.30.0.md

```diff
@@ -0,0 +1,123 @@
+k6 v0.30.0 is here! :tada: It was a bit of a slow after-holiday release, but it still packs a few major new features and improvements that users have been requesting for a long time!
+
+## New features
+
+### Shared memory read-only arrays between VUs ([#1739](https://github.com/loadimpact/k6/pull/1739))
+
+k6 has long had an issue with the handling of big data files with test fixtures. For example, if you have a huge `users.json` file with test users for your application:
+```js
+[
+  {"username": "user1", "password": "password1", "other": "some-long-data-....-1"},
+  {"username": "user2", "password": "password2", "other": "some-long-data-....-2"},
+  // ... ~1 million more users or more... :D
+  {"username": "user999999", "password": "password999999", "other": "some-long-data-....-999999"}
+]
+```
+
+If you just use `JSON.parse(open('users.json'))` in your script, then every VU will have a copy of the whole huge data set. Every VU in k6 is a separate JavaScript runtime, so there wasn't a thread-safe way to share data between them. Until now, that is!
+
+We've added a new built-in `SharedArray` object in the new `k6/data` module that allows VUs to share read-only data:
+```js
+import { SharedArray } from 'k6/data';
+import { sleep } from 'k6';
+import http from 'k6/http';
+
+let users = new SharedArray('someName', function () {
+    // This function will be called only once, in the first init context
+    // execution. Every other VU will just get a memory-safe read-only reference
+    // to the already opened data.
+    console.log('Loading users.json, this happens only once...');
+    // You are not restricted to JSON, you can do anything - parse a CSV or XML
+    // file, generate random data, etc. - as long as you return an array.
+    return JSON.parse(open('users.json'));
+});
+
+export let options = { vus: 10, duration: '30s' };
+export default function () {
+    let randomUserID = Math.floor(Math.random() * users.length);
+    let user = users[randomUserID]; // alternativaly, we can also use __VU and/or __ITER
+    console.log(`VU ${__VU} is running iteration ${__ITER} with user ${user.username}...`);
+    http.post('https://httpbin.test.k6.io/post', JSON.stringify(user));
+    sleep(Math.random() * 2); // or, better yet, use arribal-rate
```

Comment:
```suggestion
    sleep(Math.random() * 2); // or, better yet, use arrival-rate
```
also ... maybe just use arrival rate ?

Reply:
that was my initial idea, but it makes the script longer and takes a bit of focus away from the actual feature

Reply:
resolved by https://github.com/loadimpact/k6/pull/1809/commits/78875f72a82841ecb179455f9c891c87526a5424?